crates/sui-analytics-indexer/src/lib.rs

```diff
@@ -133,9 +133,12 @@ pub struct JobConfig {
     /// Maximum number of checkpoints to queue in memory.
     #[serde(default = "default_data_limit")]
     pub data_limit: usize,
-    /// Remote object store path prefix to use while writing
+    /// Remote store URL.
     #[serde(default = "default_remote_store_url")]
     pub remote_store_url: String,
+    /// Remote object store options
+    #[serde(default)]
+    pub remote_store_options: Vec<(String, String)>,
```

Comment:
It may be nice to have a bit more details on what these options are, i take it its just a set of key/value pairs?